.github/workflows/ci-agent.yml

```diff
@@ -2,6 +2,8 @@ name: pmm-agent CI
 
 on:
   push:
+    paths-ignore:
```

Comment:
Would using `paths` instead be a scalable approach?

Reply:
We need to run top level CI as well. `path-ignore` is the simplest way to do it.